plugins/modules/ds8000_assign_host_port.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2021, Matan Carmeli <matan.carmeli7@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+author: Matan Carmeli (@matancarmeli7)
+module: ds8000_assign_host_port
+short_description: Assign host ports to a DS8000 host.
+description:
+  - Assign host ports to a DS8000 host.
+options:
+  name:
+    description:
+      - The Name of the DS8000 host to work with.
+    type: str
+  host_port:
+    description:
+    - list of host_ports to assign to a specific host in DS8000 storage.
+    type: list
+    elements: str
+  force:
+    description:
+      - If set to True, in a case that a host port is assigned to another host, it will transfer it to the host that specified.
+      - If set to False, in case that a host port is assigned to another host, it won't transfer it to the host that specified.
+    default: False
+    type: bool
+extends_documentation_fragment:
+  - ibm.ds8000.ds8000.documentation
+'''
+
+EXAMPLES = r'''
+- name: Assign some host ports to host with force on false
+  ibm.ds8000.ds8000_assign_host_port:
+    hostname: "{{ ds8000_host }}"
+    username: "{{ ds8000_username }}"
+    password: "{{ ds8000_password }}"
+    name: host_name_test
+    host_port:
+      - 10000000C9A1BAB2
+      - 10000000C9A1BAB3
+
+- name: Assign some host ports to host with force on true
+  ibm.ds8000.ds8000_assign_host_port:
+    hostname: "{{ ds8000_host }}"
+    username: "{{ ds8000_username }}"
+    password: "{{ ds8000_password }}"
+    name: host_name_test
+    host_port: 10000000C9A1BAB2
+'''
+
+RETURN = r''' # '''
+
+import json
+
+try:
+    import pyds8k.exceptions
+except ImportError:
+    pass
+
+from ansible.module_utils._text import to_native
+from ansible.module_utils.basic import AnsibleModule
+
+from ..module_utils.ds8000 import (PyDs8k, costume_get_request,
+                                   ds8000_argument_spec)
+
+
+class PyDs8kHelper(PyDs8k):
+    def __init__(self, module):
+        super(PyDs8kHelper, self).__init__(module)
+        self.changed = False
+        self.failed = False
+
+    def _ds8000_assign_host_port(self, host_port):
```

Comment:
1. Use ds8000 or ds8K, but keep it consistent all over the project
2. Remove the result dictionary creation. return it without saving it as a local variable.